node-js-basics-101/node_modules/ws/lib/Constants.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->